kolibri/plugins/demo_server/assets/src/views/DemoServerIndex.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+
+  <div
+    class="banner"
+    :tabindex="0"
+    role="dialog"
+  >
+    <div class="banner-inner">
+      <div v-if="!bannerClosed">
+        <h1>Welcome to the Kolibri demo site!</h1>
+
+        <p>
+          You are welcome to explore with any of the
+          three primary user types (login credentials are in parentheses):
+        </p>
+        <ul>
+          <li>Learner (learnerdemo/pass or access as a guest)</li>
+          <li>Coach (coachdemo/pass)</li>
+          <li>Admin (admindemo/pass)</li>
+        </ul>
+
+        <p>
+          This online demo site includes the features and functionalities of the latest version,
+          but only holds a sample of the content that has been made available for offline use
+          in Kolibri.
+          It is intended for demonstration purposes and any user data on here may be cleared
+          periodically.
+        </p>
+
+        <p>
+          To learn more about using Kolibri, take a look at our
+          <a href="https://learningequality.org/documentation/">
+            documentation guides
+          </a> and
+          <a href="https://learningequality.org/download/">
+            download the latest release
+          </a> to better understand the platform in an offline context.
+        </p>
+      </div>
+      <div v-else>
+        <h1 class="closed-text">Welcome to the Kolibri demo site!</h1>
+      </div>
+      <KButton
+        class="close-button"
+        :text="bannerClosed ? 'Open' : 'Close'"
```

Comment:
maybe “more info” instead of “open”?

right now it's confusing because it says 

> Welcome to the Kolibri demo site! <kbd>OPEN</kbd>